scipy/conftest.py

```diff
@@ -151,6 +151,10 @@ def check_fpu_mode(request):
 
 array_api_compatible = pytest.mark.parametrize("xp", xp_available_backends.values())
 
+skip_array_api_invalid_arg = pytest.mark.skipif(SCIPY_ARRAY_API,
```

Comment:
can we rename this to `skip_xp_invalid_arg`, it's quite long and that is just as clear IMO.

Reply:
Fine with me. @rgommers?

Reply:
Renamed